systran_storages/storages/corpus.py

```diff
@@ -409,24 +409,35 @@ def push_file(self, local_path, remote_path):
                 raise ValueError(
                     'Cannot push %s, only support format of the corpus (application/x-tmx+xml, '
                     'text/bitext)' % local_path)
+            source = lp.get('source', '')
```

Comment:
What if: `source` or `targets` is really empty?  
We get always error, or CM2 will ignore this?

Reply:
If source and/or target are empty CM2 will ignore the check of lp